packages/synapse-react-client/src/components/Evaluation/EvaluationRoundEditor.tsx

```diff
@@ -257,28 +268,46 @@ export const EvaluationRoundEditor: React.FunctionComponent<
 
             <Row>
               <Col>
-                <CalendarWithIconFormGroup
-                  value={startDate}
-                  setterCallback={setStartDate}
+                <DateTimePicker
                   label="Round Start"
-                  isValidDate={disallowDatesBeforeNow}
-                  disabled={dayjs().isSameOrAfter(
-                    evaluationRoundInput.roundStart,
-                  )}
+                  value={startDate}
+                  onChange={setStartDate}
+                  onError={error => setStartDateError(error)}
+                  disabled={disableStartInput}
+                  // Don't allow the value to be a past date only if the input is not disabled.
+                  // i.e. allow a past date if the round has already started
+                  disablePast={!disableStartInput}
+                  slotProps={{
+                    textField: {
+                      helperText: startDateError
+                        ? 'Start date cannot be in the past'
+                        : undefined,
+                    },
+                  }}
                 />
               </Col>
               <Col>
-                <CalendarWithIconFormGroup
-                  value={endDate}
+                <DateTimePicker
                   label="Round End"
-                  setterCallback={setEndDate}
-                  isValidDate={disallowDatesBeforeNow}
+                  value={endDate}
+                  onChange={setEndDate}
+                  onError={error => setEndDateError(error)}
+                  disablePast={
+                    !dayjs(endDate).isSame(evaluationRoundInput.roundEnd)
+                  }
+                  slotProps={{
+                    textField: {
+                      helperText: endDateError
+                        ? 'End date cannot be in the past'
+                        : undefined,
+                    },
+                  }}
                 />
               </Col>
             </Row>
             <Row>
               <Col>
-                <h5>SUBMISSION LIMITS</h5>
+                <h5 style={{ marginTop: '20px' }}>SUBMISSION LIMITS</h5>
```

Comment:
Quick margin fix. Ideally would replace all of these Bootstrap components with MUI